app/assets/javascripts/discourse/tests/acceptance/topic-discovery-tracked-test.js

```diff
@@ -0,0 +1,280 @@
+import { settled, visit } from "@ember/test-helpers";
+import I18n from "I18n";
+import { test } from "qunit";
+
+import {
+  acceptance,
+  publishToMessageBus,
+  query,
+} from "discourse/tests/helpers/qunit-helpers";
+import Site from "discourse/models/site";
+import { NotificationLevels } from "discourse/lib/notification-levels";
+import topicFixtures from "discourse/tests/fixtures/discovery-fixtures";
+import { cloneJSON } from "discourse-common/lib/object";
+
+acceptance("Topic Discovery Tracked", function (needs) {
+  needs.user({
+    tracked_tags: ["tag1"],
+    watched_tags: ["tag2"],
+    watching_first_post_tags: ["tag3"],
+  });
+
+  needs.pretender((server, helper) => {
+    server.get("/c/:category-slug/:category-id/l/latest.json", () => {
+      return helper.response(cloneJSON(topicFixtures["/latest.json"]));
+    });
+
+    server.get("/tag/:tag_name/notifications", () => {
+      return helper.response({
+        tag_notification: { id: "test", notification_level: 2 },
+      });
+    });
+
+    server.get("/tag/:tag_name/l/latest.json", () => {
+      return helper.response(cloneJSON(topicFixtures["/latest.json"]));
+    });
+  });
+
+  test("visit discovery pages with tracked filter", async function (assert) {
+    const categories = Site.current().categories;
+    const category = categories.at(-1);
+    category.set("notification_level", NotificationLevels.TRACKING);
+
+    this.container.lookup("topic-tracking-state:main").loadStates([
+      {
+        topic_id: 1,
+        highest_post_number: 1,
+        last_read_post_number: null,
+        created_at: "2022-05-11T03:09:31.959Z",
+        category_id: category.id,
+        notification_level: null,
+        created_in_new_period: true,
+        unread_not_too_old: true,
+        treat_as_new_topic_start_date: "2022-05-09T03:17:34.286Z",
+      },
+      {
+        topic_id: 2,
+        highest_post_number: 12,
+        last_read_post_number: 11,
+        created_at: "2020-02-09T09:40:02.672Z",
+        category_id: category.id,
+        notification_level: 2,
```

Comment:
Is it possible to use `NotificationLevels` here instead of hardcoding the level value?

(same goes to to the other tests where you're loading state into TTS)